packages/expo-file-system/src/next/FileSystem.types.ts

```diff
@@ -146,3 +146,71 @@ export declare class File {
    */
   md5: string | null;
 }
+
+export declare class Paths {
+  /*
+   * A property containing the document directory – a place to store files that are safe from being deleted by the system.
+   */
+  static get document(): Directory;
```

Comment:
- I feel like they should be called `documentDirectory`/`cacheDirectory`, for clarity on what is returned
- What do you think about moving them to static properties of `Directory` class instead? We should unify that with #31525 🤔

Reply:
More context here: https://swmansion.slack.com/archives/C07LKHYST5H/p1726062497592479 – I think I like what @lukmccall suggested as well, imo `new Directory(Directory.cacheDirectory, "some", "path")` is very verbose and unclear, over `new Directory(Paths.cache, "some", "path")`